### PR TITLE
add payment methods

### PR DESCRIPTION
**Pull request by GeoDpto**

### CODE REVIEW

File: data/payment_services.json [hunk 1]
```diff
@@ -3598,5 +3598,86 @@
     ],
     "amount_min": 0.01,
     "amount_max": 99999.00
+  },
+  {
+    "code": "paysafe_usd",
```
Comment (Pashkalab) [inline data/payment_services.json]: paysafe_card_usd

File: data/payment_services.json [hunk 2]
```diff
@@ -3598,5 +3598,86 @@
...
+        "regexp": "\/^[_a-zA-Z0-9-]+(\\.[_a-zA-Z0-9-]+)*@[a-zA-Z0-9-]+(\\.[a-zA-Z0-9-]+)*(\\.[a-zA-Z]{2,3})$\/",
+        "required": true,
+        "position": 1
+      }
+    ],
+    "amount_min": 1,
+    "amount_max": 50000
+  },
+  {
+    "code": "paysafe_eur",
```
Comment (Pashkalab) [inline data/payment_services.json]: paysafe_card_eur_hpp

File: data/payment_services.json [hunk 3]
```diff
@@ -3598,5 +3598,86 @@
...
+        "regexp": "\/^[_a-zA-Z0-9-]+(\\.[_a-zA-Z0-9-]+)*@[a-zA-Z0-9-]+(\\.[a-zA-Z0-9-]+)*(\\.[a-zA-Z]{2,3})$\/",
+        "required": true,
+        "position": 1
+      }
+    ],
+    "amount_min": 1,
+    "amount_max": 50000
+  },
+  {
+    "code": "paysafe_inr",
```
Comment (Pashkalab) [inline data/payment_services.json]: paysafe_card_inr_hpp

File: data/payout_methods.json [hunk 4]
```diff
@@ -21175,7 +21175,7 @@
     "name": {
       "en": "Zantel ezy Pesa",
       "ru": "Zantel ezy Pesa",
-      "uk": "Zantel ezy Pesa"
+      "uk": "Zantel ezy Pes
```
Comment (MichaelKuzyk) [inline data/payout_methods.json]: поломал

**Pull request merged**